src/registry/otlp/otlp_ingester.rs

```diff
@@ -47,9 +47,43 @@ impl OtlpIterator {
 
     fn fill_buffer_from_request(&mut self, request: OtlpRequest) -> Option<usize> {
         match request {
-            OtlpRequest::Logs(_logs) => {
-                // TODO Implement the checking logic for logs
-                Some(0)
+            OtlpRequest::Logs(logs) => {
+                for resource_log in logs.resource_logs {
+                    if let Some(resource) = resource_log.resource {
+                        let mut sample_resource = SampleResource {
+                            attributes: Vec::new(),
+                            live_check_result: None,
+                        };
+                        for attribute in resource.attributes {
+                            sample_resource
+                                .attributes
+                                .push(sample_attribute_from_key_value(&attribute));
+                        }
+                        self.buffer.push(Sample::Resource(sample_resource));
+                    }
+
+                    for scope_log in resource_log.scope_logs {
+                        if let Some(scope) = scope_log.scope {
+                            for attribute in scope.attributes {
+                                self.buffer.push(Sample::Attribute(
+                                    sample_attribute_from_key_value(&attribute),
+                                ));
+                            }
+                        }
+
+                        for log_record in scope_log.log_records {
+                            // Only process log records that have event_name set.
+                            // Per the OTLP spec: "Presence of event_name on the log record identifies this record as an event."
+                            if log_record.event_name.is_empty() {
+                                continue;
```

Comment:
I think it makes sense to live-check logs in custom policies. E.g. one can check that all logs are structured and/or certain logs that have specific attributes also have other attributes (e.g. if you have `exception.message` you must also have `exception.type`).

There won't be a reference definition to validate against *unless there is an event name)